src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -9,8 +9,12 @@
     <AssemblyOriginatorKeyFile>..\..\buildscripts\CastleKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Core.Tests\Resources\file1.txt" />
-    <EmbeddedResource Include="Core.Tests\Resources\MoreRes\file2.txt" />
+    <EmbeddedResource Include="Core.Tests\Resources\file1.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Core.Tests\Resources\MoreRes\file2.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Makes sense now, `AssemblyResourceFactoryTestCase` and `FileResourceFactoryTestCase` both use those files from different places.

My guess for why it works with the project.json tooling is that the `Directory.GetCurrentDirectory()` returns where the project.json file is, MSBuild stuff always runs out of bin.